test-jest/specs/chart/PieChart.spec.tsx

```diff
@@ -20,7 +12,7 @@ describe.only('<PieChart />', () => {
     { name: 'Group F', value: 189 },
   ];
 
-  it('Renders 6 sectors circles in simple PieChart', () => {
+  test('Renders 6 sectors circles in simple PieChart', () => {
```

Comment:
It is just an alias for test. Thus this does not make a difference.

Reply:
yep! but as a dev I'd rather stick to one or the other. Just a preference change